src/docs/automation-and-resiliency/openshift-project-resource-quotas.md

```diff
@@ -27,7 +27,7 @@ New project sets provisioned in **all clusters** of the BC Gov Private Cloud Paa
 
 **Note**: The Platform Services team may approach your team to discuss a downgrade in your quota if they find your application isn't using all of the resources within the current quota.
 
-If the default allocations aren't sufficient for your application, [you can ask for a quota increase](/request-quota-increase-for-openshift-project-set/). You'll need a Sysdig dashboard that shows that your application needs more of a specific resource type (CPU, RAM or storage) in a specific namespace. Provide this proof to the Platform Services team before they can approve a quota increase that you submit in the [Platform Project Registry](https://registry.developer.gov.bc.ca/public-landing).
+If the default allocations aren't sufficient for your application, [you can ask for a quota increase](/request-quota-increase-for-openshift-project-set/). You'll need a Sysdig dashboard that shows that your application needs more of a specific resource type (CPU, RAM or storage) in a specific namespace. Provide this proof to the Platform Services team before they can approve a quota increase that you submit in the [Platform Product Registry](https://registry.developer.gov.bc.ca/login).
```

Comment:
I dont think registry has login page

Reply:
My bad, now it get redirect...But I still think just `https://registry.developer.gov.bc.ca/` would be good enough, what you think @ShellyXueHan

Reply:
either way works fine, but if we are just referencing the product registry, might be better to using `https://registry.developer.gov.bc.ca/` without the subpath, incase our developers decide to change it later on then it'll throw a 404 ;)

Reply:
Perfect, thank you team, I will update it to make it more clear! Thank you both so much!